bundle/manifests/mcg.openshift.io_managedmcgs.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.7.0
+    controller-gen.kubebuilder.io/version: v0.4.1
```

Comment:
avoid downgrading version

Reply:
This is the version that the controller-gen is pinned to in the repository (which was tested with the current version of operator sdk and was automatically generated by the same): https://github.com/red-hat-storage/mcg-osd-deployer/blob/main/Makefile#L181

Reply:
update the controller-gen version to latest

Reply:
Why? That might cause the addon to exhibit unexpected behavior in the future. PTAL my comment above: **This is the version that the controller-gen is pinned to in the repository (which was tested with the current version of operator sdk and was automatically generated by the same)**

Noobaa still uses a much older version of Operator SDK https://github.com/noobaa/noobaa-operator/blob/master/Makefile#L32 (thus the deps coming with it), because that's what the other deps being used there are tested out with. We cannot update a dependency without first properly testing all cases and whether it introduces a side-effect or not.

Reply:
we were testing addon with this controller-gen version in the cluster, we are using v0.4.1 version in local testing that is because the controller-gen version in your local machine is v0.4.1

Reply:
> because the controller-gen version in your local machine is v0.4.1

Yes, because I assumed that was the agreed-upon version when the `controller-gen` PR was reviewed. This is not only the one used in master, but also on the CI. Nonetheless since we are already using this for a while now (as you pointed out), I'll bump this to `v0.7.0`.

Reply:
Upgraded and pushed: https://github.com/red-hat-storage/mcg-osd-deployer/pull/38/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52